build.gradle

```diff
@@ -14,17 +14,24 @@ ext {
     opentelemetryProjects = subprojects - project(":opentelemetry-bom")
 }
 
-release {
-    defaultVersionStrategy = Strategies.getSNAPSHOT()
-}
+// Nebula plugin will not configure if .git doesn't exist, let's allow building on it by stubbing it
+// out.
+def releaseTask
+if (file('.git').exists()) {
```

Comment:
This looks a bit brittle. I think something like `project.hasProperty('release')` would be better.

Reply:
I think we'll always have .git - on the flip side, I used this check because I want to explicitly handle it being missing, without accidentally handling a case where `.git` is actually there and `release` isn't for whatever reason which isn't ever expected.

Reply:
I just checked and it is at least there in a `git worktree`. I guess it is also there when checked out as `git submodule` so probably you are right. But there might be cases I haven't thought of :smiley: